components/logins/src/schema.rs

```diff
@@ -224,7 +228,15 @@ fn upgrade(_db: &Connection, from: i64) -> Result<()> {
         "Upgrading from user_version = 0 should already be handled (in `init`)"
     );
 
-    // Schema upgrades that should happen after the sqlcipher -> plaintext migration go here
+    // Schema upgrades.
+    if from == 1 {
+        // Just one new nullable column makes this fairly easy
+        db.execute_batch("ALTER TABLE loginsM ADD enc_unknown_fields TEXT;")?;
+    }
+    // XXX - next migration, be sure to:
+    // from = 2;
+    // if from == 2 ...
+    db.execute_batch(&SET_VERSION_SQL)?;
```

Comment:
we really should use our sql_support code for migrating etc, but I didn't want to block this on that especially while we still support sqlcipher migrations.